.github/workflows/integration-test.yml

```diff
@@ -48,3 +48,47 @@ jobs:
 
       - name: Integration Test
         run: make integration_test
+
+jobs:
```

Comment:
https://github.com/beyondstorage/go-service-ftp/actions/runs/1260281105

I think it's invalid to declare two `jobs` in the same file. Instead, we need to:

```yaml
jobs:
  fclairamb_ftpserver:
    name: xxxx
  asf_mina-ftpserver:
    name: xxxx
```